src/main/java/org/kohsuke/github/extras/authorization/JWTTokenProvider.java

```diff
@@ -173,16 +172,16 @@ private String refreshJWT() {
 
         // Let's set the JWT Claims
         JwtBuilder builder = Jwts.builder()
-                .setIssuedAt(Date.from(issuedAt))
-                .setExpiration(Date.from(expiration))
-                .setIssuer(this.applicationId)
-                .signWith(privateKey, SignatureAlgorithm.RS256);
+                .issuedAt(Date.from(issuedAt))
+                .expiration(Date.from(expiration))
+                .issuer(this.applicationId)
+                .signWith(privateKey, Jwts.SIG.RS256);
```

Comment:
Suggest something like this to deal with backward compatibility:
```
        JwtBuilder builder = Jwts.builder();

        // Let's set the JWT Claims
        try {
            builder = builder.issuedAt(Date.from(issuedAt))
                    .expiration(Date.from(expiration))
                    .issuer(this.applicationId)
                    .signWith(privateKey, Jwts.SIG.RS256);
        catch (java.lang.NoSuchMethodError e) { 
            builder = builder.setIssuedAt(Date.from(issuedAt))
                    .setExpiration(Date.from(expiration))
                    .setIssuer(this.applicationId)
                    .signWith(privateKey, SignatureAlgorithm.RS256);
        }
```
This will need manual testing with the old version to verify it works.